src/datastore/pg-store.ts

```diff
@@ -396,6 +402,76 @@ export class PgStore {
     });
   }
 
+  async getBlocksWithMetadata({ limit, offset }: { limit: number; offset: number }) {
+    return await this.sql.begin(async sql => {
+      // get block list
+      const { results: blocks, total: block_count } = await this.getBlocks({ limit, offset });
+      const blockHashValues: Buffer[] = [];
+      const indexBlockHashValues: Buffer[] = [];
+      blocks.forEach(block => {
+        const indexBytea = hexToBuffer(block.index_block_hash);
```

Comment:
Are these conversions necessary? Doesn't the new pg lib handle both hex strings and buffers automatically now?

Reply:
It works with `IN` but not with `ANY` clause. I have removed the conversions since we're using `IN` clause anyways.